### DISCUSSION TITLE
Critical features of a successful web app?

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#41] [Ideas]
Some of the must have features of most web applications are:
Subscription based billing
Social Logins (Linkedin, Twitter, etc)
Live Chat
User Rights Management
Cybersecurity
Search Engine Optimization

**Comment by kentcdodds** [comment]
- Subscription based billing: We'll have docs on Stripe usage in the future
- Social Logins (Linkedin, Twitter, etc): This will be added in the future
- Live Chat: We'll have docs on this in the future
- User Rights Management: This is already implemented in the data model and just needs some UI for managing this.
- Cybersecurity: Not sure how this is a "feature" but we will add support for a content security policy in the future.
- Search Engine Optimization: We have built-in support for some important aspects of SEO (like meta tags), but this isn't something you can really build into a project starter
